geoindicators/src/test/resources/org/orbisgis/orbisprocess/geoclimate/geoindicators/lczTests/expectedLcz.geojson

```diff
@@ -3,6 +3,30 @@
 "name": "expectedLcz",
 "crs": { "type": "name", "properties": { "name": "urn:ogc:def:crs:EPSG::2154" } },
```

Comment:
This data is for integration tests. It must be out of the code source otherwise it becomes complicated to download the sources from git

Reply:
I do not understand. I do not think it is for integration test (while it was written above the test that uses this file). In my opinion it is used as a test on the LczClassification algorithm, thus should remain in the source. I am actually working on creating a restricted dataset from the bdtopodata instead of saving locally all bdtopo results.

Reply:
Would be better to talk in live